src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -3,6 +3,25 @@
 {% set title = _('Status & Versions') %}
 {% block title %}{{ dev_page_title(title, addon) }}{% endblock %}
 
+{% if addon.has_listed_versions() %}
+  {% set has_listed_versions = True %}
+{% endif %}
+{% if addon.has_unlisted_versions() %}
+  {% set has_unlisted_versions = True %}
+{% endif %}
+
+{% if has_listed_versions and has_unlisted_versions %}
+  {% macro distro_tag(channel) %}
```

Comment:
yep, that's what the UX mocks that Jorge agreed to specified - that the tags _only_ appear when the add-on has mixed listed and unlisted versions.